src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -290,26 +292,24 @@ public void SetSubEditorSettingsForMapping(ColumnMappingDTO model)
          else
          {
             var errorColumnDTO = _mappings.FirstOrDefault(c => c.ColumnInfo?.RelatedColumnOf == model.MappingName);
-            var errorColumn = ((MappingDataFormatParameter)errorColumnDTO?.Source)?.MappedColumn;
-            if (errorColumn?.Unit != null && !errorColumn.Unit.ColumnName.IsNullOrEmpty())
-            {
+            var errorColumn = ((MappingDataFormatParameter) errorColumnDTO?.Source)?.MappedColumn;
+
+            if (errorColumn?.Unit != null && !errorColumn.Unit.ColumnName.IsNullOrEmpty()) 
                columns.Add(errorColumn.Unit.ColumnName);
-            }
 
             dimensions.AddRange(_columnInfos
```

Comment:
Here. THe ist can potentially be empty. Then the next call to setUnitOPtiosn makes a First and this could crash

Reply:
all the other changes in this file are because of wrong formatting.